src/org/omegat/Bundle_ja.properties

```diff
@@ -83,1224 +83,1255 @@ app-version-template-pretty={0} {1}
+TF_INTRO_EMPTYPROJECT=\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u306b\u7ffb\u8a33\u53ef\u80fd\u306a\u30d5\u30a1\u30a4\u30eb\u304c\u542b\u307e\u308c\u3066\u3044\u306a\u3044\u3088\u3046\u3067\u3059\u3002\n\n\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u306b\u30d5\u30a1\u30a4\u30eb\u3092\u8ffd\u52a0\u3059\u308b\u306b\u306f:\n\n\t- \u3053\u306e\u30a8\u30c7\u30a3\u30bf\u30da\u30a4\u30f3\u306b\u30d5\u30a1\u30a4\u30eb\u3092\u30c9\u30e9\u30c3\u30b0\u30a2\u30f3\u30c9\u30c9\u30ed\u30c3\u30d7\u3057\u307e\u3059\u3002\n\t-[\u30d7\u30ed\u30b8\u30a7\u30af\u30c8]>[\u539f\u6587\u30d5\u30a1\u30a4\u30eb\u3092\u8ffd\u52a0...]\u3092\u4f7f\u7528\u3057\u307e\u3059\u3002\n\t-\u300c\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u30d5\u30a1\u30a4\u30eb\u300d\u30a6\u30a3\u30f3\u30c9\u30a6\u3067\u300c\u539f\u6587\u30d5\u30a1\u30a4\u30eb\u3092\u8ffd\u52a0...\u300d\u3092\u30af\u30ea\u30c3\u30af\u3057\u307e\u3059\u3002\n\t-\u30d5\u30a1\u30a4\u30eb\u30de\u30cd\u30fc\u30b8\u30e3\u3092\u4f7f\u7528\u3057\u3066\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u306e 'source' \u30d5\u30a9\u30eb\u30c0\u306b\u30d5\u30a1\u30a4\u30eb\u3092\u30b3\u30d4\u30fc\u3057\u3001\n\t   [\u30d7\u30ed\u30b8\u30a7\u30af\u30c8]>[\u518d\u8aad\u307f\u8fbc\u307f]\u3067\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u3092\u518d\u8aad\u307f\u8fbc\u307f\u3057\u307e\u3059\u3002\n\n\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u306b\u30d5\u30a1\u30a4\u30eb\u304c\u542b\u307e\u308c\u3066\u3044\u3066\u3082\u3001\u305d\u306e\u5185\u5bb9\u304c\u3053\u3053\u306b\u8868\u793a\u3055\u308c\u306a\u3044\u5834\u5408:\n\n\t- verify that their format is supported by OmegaT:\n\t  if local filter settings are enabled\n\t  use Project > Properties... > Local File Filters...,\n\t  otherwise use Options > Global File Filters...;\n\t- open them in their original application and verify that they are not empty.
```

Comment:
I can't get a diff that shows me anything other than the escaped Unicode strings, so I can't comment on the specifics, but:

"verify that their format..." の部分は、敢えて英語のままに残していますか。

Reply:
The change is happened because a resource bundle plugin in OmegaT 6.1.0 changes the logic to produce the escape. 
"Verify that..." is a missing translation part of the L10N/ja project, so I'd like to update in the project L10N/ja and update the PR here.